keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -92,17 +95,7 @@ final {
         throw std::invalid_argument("file is corrupt(truncated)");
       }
 
-      boost::interprocess::map_options_t map_options = boost::interprocess::default_map_options;
-
-#ifdef MAP_HUGETLB
-      map_options |= MAP_HUGETLB;
-#endif
-
-      if (!load_lazy) {
-#ifdef MAP_POPULATE
-        map_options |= MAP_POPULATE;
-#endif
-      }
+      boost::interprocess::map_options_t map_options = internal::MemoryMapFlags::FSAGetMemoryMapOptions(loading_strategy);
```

Comment:
In general it's a good practice to declare variables `const`, if they are not gonna to change. I think here we can do so.